docs/src/main/asciidoc/api/account-api.adoc

```diff
@@ -0,0 +1,179 @@
+
+=== Account Resource URI
+
+*/2012-04-24/Accounts/\{AccountSid}*
+
+=== Resource Properties
+
+[cols=",",options="header",]
+|=========================================================================================================================
+|Property |Description
+|Sid |A string that uniquely identifies this account.
+|DateCreated |The date that this account was created.
+|DateUpdated |The date that this account was last updated.
+|FriendlyName |A description of this account, up to 64 characters long. By default the FriendlyName is your email address.
+|Status |The status of this account. Possible values are active, suspended, and closed.
+|AuthToken |The authorization token for this account. This should not be shared.
+|Uri |The URI for this account, relative to http://localhost:port/restcomm.
+|=========================================================================================================================
+
+=== Supported Operations
+
+*HTTP GET.* Returns the representation of an Account resource, including the properties above.
+
+* Account Resource URI. */2012-04-24/Accounts/\{EmailAddress}*
+
+**HTTP POST/PUT**. Modifies an Account resource and returns the representation, including the properties above. Below you will find a list of optional parameters.
+
+=== Request Parameters
+
+[cols=",",options="header",]
+|==============================================================================================
+|Parameter |Description
+|FriendlyName |A description of this account, up to 64 characters long.
+|Status |The status of this account. Possible values are active, suspended, and closed.
+|Password |A password that will be used to generate the AuthToken for the new Account resource.
+|==============================================================================================
+
+**Get information about the default account.**
+
+....
+curl -X GET  http://ACae6e420f425248d6a26948c17a9e2acf:77f8c12cc7b8f8423e5c38b035249166@127.0.0.1:8080/restcomm/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf
+....
+
+
+**Change default account password(AuthToken).**
+
+....
+curl -X PUT http://ACae6e420f425248d6a26948c17a9e2acf:77f8c12cc7b8f8423e5c38b035249166@127.0.0.1:8080/restcomm/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf -d "Password=NewPassword"
+....
+
+NOTE: The above command uses the Account SID and the one below uses the Email Account. Note the administrator%40company.com is used instead of administrator@company.com . This is because using curl on the bash terminal doesn't parse the @ correctlyl. If you were to running on a browser, you can safely use the @ as the web browser will correctly handle it.
+
+....
+curl -X GET  http://administrator%40company.com:77f8c12cc7b8f8423e5c38b035249166@127.0.0.1:8080/restcomm/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf
+....
+
+The above commands will print an output similar to the one below:
+
+----
+<RestcommResponse>
```

Comment:
The restcomm response below doesn't seem correct as the account in the sbc project doesn't have any notion of AvailablePhoneNumbers, IncomingPhoneNumbers, etc...